stream-product-catalog/product-catalog-core/src/main/java/com/backbase/stream/productcatalog/configuration/ProductCatalogServiceConfiguration.java

```diff
@@ -19,7 +18,6 @@
  */
 @Configuration
 @EnableConfigurationProperties(BackbaseStreamConfigurationProperties.class)
-@Import(DbsWebClientConfiguration.class)
```

Comment:
Getting an error generating this bean:
>[11:38:23.054]               [main] [WARN] [org.springframework.boot.web.servlet.context.AnnotationConfigServletWebServerApplicationContext] Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'ingestionController' defined in file [/Users/danielb.moran@bnc.ca/IdeaProjects/bne/dis/ingestion-dbs/target/classes/ca/bnc/backbase/controller/IngestionController.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'ingestUnitOfWorkExecutorServiceImpl' defined in file [/Users/danielb.moran@bnc.ca/IdeaProjects/bne/dis/ingestion-dbs/target/classes/ca/bnc/backbase/service/impl/IngestUnitOfWorkExecutorServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'productCatalogService' defined in class path resource [com/backbase/stream/productcatalog/configuration/ProductCatalogServiceConfiguration.class]: Unsatisfied dependency expressed through method 'productCatalogService' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'reactiveProductCatalogService' defined in class path resource [com/backbase/stream/productcatalog/configuration/ProductCatalogServiceConfiguration.class]: Unsatisfied dependency expressed through method 'reactiveProductCatalogService' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'accountPresentationClient' defined in class path resource [com/backbase/stream/productcatalog/configuration/ProductCatalogServiceConfiguration.class]: Unsatisfied dependency expressed through method 'accountPresentationClient' parameter 0; nested exception is org.springframework.beans.factory.NoUniqueBeanDefinitionException: No qualifying bean of type 'org.springframework.web.reactive.function.client.WebClient' available: expected single matching bean but found 2: accessProviderWebClient,interServiceWebClient

Reply:
the DbsWebClientConfiguration was removed in favour of the m10y libraries and the ssdk web client, however as m10y was unsupported we had to move it back, so this guy also needs to be back again. nice catch!

Reply:
actually, this is different, I'm fixing it now